src/components/components.cpp

```
@@ -65,17 +65,18 @@ pgr_connectedComponents(pgrouting::UndirectedGraph &graph) {
     results[components[vd]].push_back(graph[vd].id);
   }
 
-  return detail::componentsResult(results);
+  return results;
 }
 
 //! Strongly Connected Components Vertex Version
-std::vector<II_t_rt> strongComponents(pgrouting::DirectedGraph &graph) {
+std::vector<std::vector<int64_t>>
+strongComponents(pgrouting::DirectedGraph &graph) {
```

Cppcheck (2.20.0)

[style] The function 'strongComponents' is never used.

(unusedFunction)